bedrock/products/views.py

```diff
@@ -36,6 +36,18 @@ def vpn_landing_page(request):
     return l10n_utils.render(request, template_name, context)
 
 
+@require_safe
+def vpn_desktop_page(request):
+    template_name = 'products/vpn/desktop/index.html'
+
+    context = {
+        'allowed_countries': vpn_allowed_countries(),
+        'default_monthly_price': vpn_default_monthly_price(),
+    }
+
+    return l10n_utils.render(request, template_name, context, ftl_files=['products/vpn/desktop'])
+
+
```

Comment:
I was wanting to do this programmatically, but it looks like to pass the `context` object, you have to define each individual view. Is that okay?

Reply:
Does the SEO page need geo-location and pricing functionality? If so it will need to be a view like this. If not the page could be a simpler URL without a view.

Reply:
I think we will (and if not, should) list countries at least. Pricing should just direct folks to the main page. 

However, the CTA is crafted to not need it. If we can all agree if these can be left off the SEO pages (without any SEO ramifications), then we totally can remove it.

CTA:
<img width="808" alt="image" src="https://user-images.githubusercontent.com/2692333/110394796-9d41fe00-8032-11eb-9cb4-38b84fe1c2d5.png">

Reply:
This has been revised to use the `page` utils render function.